guides/common/modules/proc_configuring-minimal-data-collection-for-hosts.adoc

```diff
@@ -0,0 +1,38 @@
+[id="configuring-minimnal-data-collection-for-hosts"]
+= Configuring Minimal data collection for hosts
+
+You can configure a host in {Project} to use minimal reporting if you want to limit the data sent to subscriptions service and disable Insights integration.
+This option allows you to limit the amount of system data sent to Red{nbsp}Hat while still supporting subscriptions service.
+
+.Prerequisites
+* Ensure existing data in Insights Inventory is deleted before switching a host to minimal reporting.
+To remove the system, go to *Inventory* > *Systems* in the {RHCloud}, select the systems, and click *Delete*.
+* You have selected an organization and location.       
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Insights* > *Inventory Upload*.
+. Select the *Minimal data collection* setting from the dropdown menu under *Settings*.
+When you select this option, {Project} also updates the corresponding setting on the *Insights* tab under *Administer* > *Settings*. 
+You can also enable this setting by navigating to *Administer* > *Settings*, and on the *Insights* tab, setting *Minimal data collection* to *Yes*. 
+When you select *Minimal data collection*, {Project} does not include hostnames or IP addresses in the report, so the settings to obfuscate this information do not apply. 
+{Project} also ignores the setting to exclude installed packages, as it always excludes them from reports at this level.
++
+[NOTE]
+====
+Switching from *Minimal data collection* to *Optional data collection* enables all of the settings. 
```

Comment:
What settings?
"all of the settings" is ambiguous.

Reply:
Changing to `available settings`.